client/blocks/eligibility-warnings/warning-list.tsx

```diff
@@ -13,54 +16,81 @@ interface ExternalProps {
 
 type Props = ExternalProps & LocalizeProps;
 
-export const WarningList = ( { context, translate, warnings, showContact = true }: Props ) => (
-	<div>
-		{ getWarningDescription( context, warnings.length, translate ) && (
-			<div className="eligibility-warnings__warning">
-				<div className="eligibility-warnings__message">
-					<span className="eligibility-warnings__message-description">
-						{ getWarningDescription( context, warnings.length, translate ) }
-					</span>
+export const WarningList = ( { context, translate, warnings, showContact = true }: Props ) => {
+	const [ isAdminStyleEnabled, setAdminStyleEnabled ] = useState( true ); // State to track toggle
+
+	// Function to handle toggle changes
+	const handleAdminStyleToggle = ( checkedValue: boolean ) => {
+		console.log( 'checkedValue', checkedValue );
+		setAdminStyleEnabled( checkedValue );
+		const siteId = getSelectedSiteId( getState() );
+		console.log( 'siteId', siteId );
+
+		if ( siteId !== null ) {
+			const adminStyle = checkedValue ? 'wp-admin' : 'calypso';
```

Comment:
Wondering if it makes sense to create constants for `wp-admin` and `calypso` as they get referenced more and more 🤔